File: src/main/java/com/morak/performancetracker/aop/rest/RestPointcut.java
package com.morak.performancetracker.aop.rest;

import java.lang.reflect.Method;
import org.springframework.aop.support.StaticMethodMatcherPointcut;

public class RestPointcut extends StaticMethodMatcherPointcut {

    @Override
    public boolean matches(Method method, Class<?> ignored) {
        return method.getName().contains("execute");
    }
}

File: src/main/kotlin/com/morak/performancetracker/aop/rest/RestPointcut.kt
package com.morak.performancetracker.aop.rest

import org.springframework.aop.support.StaticMethodMatcherPointcut
import java.lang.reflect.Method

class RestPointcut : StaticMethodMatcherPointcut() {
    override fun matches(method: Method, ignored: Class<*>): Boolean {
        return method.name.contains("execute")
    }
}
